src/fs.rs

```diff
@@ -23,8 +23,9 @@ pub fn make_temp_dir(
     Some(ref p) => p.to_path_buf(),
     None => std::env::temp_dir(),
   }.join("_");
+  let mut rng = rand::thread_rng();
```

Comment:
🤷🏻‍♂️but out of curiosity, this change? From the docs:

```
Retrieve the lazily-initialized thread-local random number generator, seeded by the system. Intended to be used in method chaining style, e.g. thread_rng().gen::<i32>().
```

Reply:
I had to downgrade rand to be compatible with other deps and the API is different.

Reply:
Ah ok got it.

Reply:
Just realized I didn't change the version in Cargo.toml... Fixed.